### PR TITLE
Code for decoding the jpeg image format

### PR DESCRIPTION
**Pull request by Suyash20012000**
This is trial code made for decoding jpeg format image

### CODE REVIEW

**Comment by Suyash20012000** [comment]
This is code for decoding the jpeg format image to be reviewed by vikram dattu

File: djpeg.c [hunk 1]
```diff
@@ -836,3 +836,61 @@ main(int argc, char **argv)
   exit(jerr.num_warnings ? EXIT_WARNING : EXIT_SUCCESS);
   return 0;                     /* suppress no-return-value warnings */
 }
+
+/* self demo code <--not part of official libjpeg */
+int decode_greyscale(unsigned char *in_buf,unsigned long *in_buf_sz)
+{
```
Comment (vikramdattu) [inline djpeg.c:842]: Why the name `decode_grayscale`?
Doesn't it decode any kind of jpeg file?

File: djpeg.c [hunk 2]
```diff
@@ -836,3 +836,61 @@ main(int argc, char **argv)
...
+  /*specify data source for decompression*/
+  jpeg_mem_src(&cinfo, &in_buf, &in_buf_sz);
+
+  /* Start decompressor */
+  (void)jpeg_start_decompress(&cinfo);
+
+
+  
+  unsigned char *buf[3];
+/* Process data */
```
Comment (vikramdattu) [inline djpeg.c:868]: minor: fix code asthetics

Alignment is wrong here

File: esp32_jpeg_enc/build/.bin_timestamp [hunk 3]
```diff
@@ -0,0 +1 @@
+80fa9f05bf90961bc834e198a7ecbe71  /home/suyash/esp/esp-idf/libjpeg-turbo-feature-esp32_build/esp32_jpeg_enc/build/jpeg-enc.bin
```
Comment (vikramdattu) [inline esp32_jpeg_enc/build/.bin_timestamp:1]: Remove build files.
Files generated under build/ folder

**Comment by vikramdattu** [comment]
Too many issues. New PR has been raised. Closing this PR.

**Pull request closed without merging**